koji-prometheus-exporter.py

```diff
@@ -345,6 +357,7 @@ def scrape():
             'koji_enabled_hosts_count': koji_enabled_hosts_count_family,
             'koji_enabled_hosts_capacity': koji_enabled_hosts_capacity_family,
             'koji_task_load': koji_task_load_family,
+            'koji_task_completed_total': koji_task_completed_total_family,
```

Comment:
Can you move this up underneath `koji_errors_total` since it is conceptually similar?

Reply:
Certainly!